backend/packages/db/ArangoDB.js

```diff
@@ -0,0 +1,272 @@
+// @flow
```

Comment:
It is a **high priority** to write integration tests for this module and open source it. This is critical code.